src/app/(routes)/across-plus/_components/mockups-carousel.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import Image from "next/image";
+import Autoscroll from "embla-carousel-auto-scroll";
+
+import { Carousel, CarouselContent, CarouselItem } from "@/app/_components/carousel";
+import mockupImage1 from "../_assets/mockup-1.png";
+import mockupImage2 from "../_assets/mockup-2.png";
+import mockupImage3 from "../_assets/mockup-3.png";
+import mockupImage4 from "../_assets/mockup-4.png";
+import carouselBgImage from "../_assets/carousel-bg.png";
+import { twMerge } from "@/app/_lib/tw-merge";
+
+const CAROUSEL_IMAGES = [mockupImage1, mockupImage2, mockupImage3, mockupImage4];
+
+export function MockupsCarousel() {
+  return (
+    <Carousel
+      className="w-full max-w-[794px]"
+      opts={{
+        loop: true,
+      }}
+      plugins={[Autoscroll()]}
+    >
+      <CarouselContent className={twMerge("mx-10 gap-2 sm:mx-40 md:mx-52")}>
+        {[...CAROUSEL_IMAGES, ...CAROUSEL_IMAGES].map((image, index) => (
+          <CarouselItem key={index}>
+            <div className="p-1">
+              <Image
+                src={image}
+                alt="mockup"
+                className="opacity-80"
+                width={360}
+                priority
+                quality={100}
+                unoptimized
```

Comment:
OOC why do we not want to optimize this next/image tag?

Reply:
I somehow can not get a crisp resolution to work without setting this flag to `true`... Any ideas?

Reply:
It's finicky especially since we're reducing the size of the image.